spec/feature/posts_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+describe 'Posts', type: :feature do
+  before(:all) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'Teacher from Mexico.', post_counter: 1)
+  end
+  it 'should have a list of posts' do
+    visit '/users/:user_id/posts'
+    expect(page).to have_content('Tom')
+    expect(page).to have_content('Number of posts: 1')
+    expect(page).to have_content('Hello')
+    expect(page).to have_content('This is my first post')
+  end
+end
```

Comment:
- [ ] As required for this project, kindly make sure all your `controllers_tests_files` are in the `spec/requests/` directory and not the `spec/feature/` directory.  So you should use `type: request` in each file instead of `type: request` as shown in the example code below:
```ruby
require 'rails_helper'

 RSpec.describe "Users", type: :request do
   ...
 end
```

Reply:
Done